sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -168,6 +169,10 @@ export interface DispositionStatusOptions extends OperationOptionsBase {
    * This should only be provided if `session` is enabled for a Queue or Topic.
    */
   sessionId?: string;
+  /**
+   * Retry options.
+   */
+  retryOptions: RetryOptions | undefined;
```

Comment:
Any reason why this is not `retryOptions?: RestryOptions` following the same patterns as others in this interface?

Reply:
I specifically don't want someone to forget to fill it out.

Reply:
I'm in favor of this pattern for non-user-facing code :smile: